src/lib.ts

```diff
@@ -108,8 +145,31 @@ export const args = Args.create("railo", "A script for farming elf stuff", {
     ],
     default: "both",
   }),
+  orb: Args.string({
+    options: [
+      ...[...Object.values(trainbots)]
+        .map((o) => Object.keys(o))
+        .flat()
+        .map((key) => [key, `Target the ${key} trainbot with the orb.`] as [string, string]),
+      ["none", "Don't use it!"],
+    ],
+    default: "none",
+  }),
 });
 
+let orbTarget: Monster | null = null;
+export function validateOrbArgument(target: string, car: string) {
+  if (target === "none") return;
+  if (!have($item`miniature crystal ball`)) return;
+  if (!(car in trainbots)) throw new Error("Invalid car specified!");
+  const carTargets = trainbots[car as keyof typeof trainbots];
+  if (!(target in carTargets)) throw new Error("Invalid target specified");
+  orbTarget = carTargets[target as keyof typeof carTargets];
```

Comment:
I don't know if this code-style level of feedback is appreciated in a PR like this but just in case it is -- having a `validate` function set a variable feels like unexpected behavior to me.

This could be addressed by renaming to `validateAndSetOrbArgument` or even just changing it to `setOrbArgument` (with the validation being a happy secondary byproduct)

While we're here I'd also mirror the naming of `getOrbTarget` and call this `setOrbTarget` / `validateAndSetOrbTarget`.